.husky/_/pre-commit

```diff
@@ -1,9 +1,2 @@
-#!/bin/sh
-
-# Run linting and formatting
-npm run lint:fix
-npm run format
-
-
-# If everything passes, allow the commit
-# If any of the commands fail, the commit will be aborted
+#!/usr/bin/env sh
```

Comment:
why did you change this file it is needed

Reply:
Actually that's because of the rebasing I did , but literally I didn't need to change that one.